### PR TITLE
issues/1933

### LINKED ISSUE #1933: Fix total donated amount value in donation method report page
## Issue Overview

Test donation total earnings   = 15.122
Offline donation total earning = 202
image

## Expected Behavior

Show total donation amount.

## Current Behavior

Show only current month donation amount.

## Possible Solution

Pass correct time to `get_earnings` function.

## Todos
- [ ] Tests
- [ ] Documentation

### PR DESCRIPTION
**Pull request by ravinderk**
## Description

This Pr will resolve #1933 

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/reports/class-gateways-reports-table.php [hunk 1]
```diff
@@ -215,7 +215,7 @@ public function reports_data() {
 				'complete_sales'  => $complete_count,
 				'pending_sales'   => $pending_count,
 				'total_sales'     => $complete_count + $pending_count,
-				'total_donations' => give_currency_filter( give_format_amount( $stats->get_earnings( 0, 0, 0, $gateway_id ), array( 'sanitize' => false ) ) ),
+				'total_donations' => give_currency_filter( give_format_amount( $stats->get_earnings( 0, strtotime('04/13/2016' ), current_time('timestamp' ), $gateway_id ), array( 'sanitize' => false ) ) ),
```
Comment (ravinderk) [inline includes/admin/reports/class-gateways-reports-table.php]: @DevinWalker I set start date to initial plugin release date to get earning.

Comment (ravinderk) [inline includes/admin/reports/class-gateways-reports-table.php]: @DevinWalker PR merge but review it once :)

Comment (DevinWalker) [inline includes/admin/reports/class-gateways-reports-table.php]: @ravinderk this isn't the correct date of Give's release. It's `2015` not `2016`.

Comment (ravinderk) [inline includes/admin/reports/class-gateways-reports-table.php]: @DevinWalker date updated:
For Ref: https://github.com/WordImpress/Give/commit/73e6520a77cf0e0d3a9c297880465afa70a71ca2

**Pull request merged**